scripts/kind-e2e/e2e.sh

```diff
@@ -280,9 +280,10 @@ function test_with_e2e_tests {
     # Setup the KUBECONFIG env
     export KUBECONFIG=$(echo ${PRJ_ROOT}/output/kind-config/dapper/kind-config-cluster{1..3} | sed 's/ /:/g')
 
-    go test -args -ginkgo.v -ginkgo.randomizeAllSpecs -ginkgo.reportPassed \
+    go test -v -args -ginkgo.v -ginkgo.randomizeAllSpecs \
         -dp-context cluster2 -dp-context cluster3  \
-        -report-dir ${DAPPER_SOURCE}/${DAPPER_OUTPUT}/junit 2>&1 | \
+	-ginkgo.noColor -ginkgo.reportPassed 
```

Comment:
you are mising a \ at the end of this line

Reply:
Thanks, fixed.